package.json

```diff
@@ -23,39 +26,71 @@
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "https://github.com/smg-automotive/fe-example-pkg.git"
+    "url": "https://github.com/smg-automotive/components-pkg.git"
   },
-  "homepage": "https://github.com/smg-automotive/fe-example-pkg#readme",
+  "homepage": "https://github.com/smg-automotive/components-pkg#readme",
   "devDependencies": {
+    "@babel/core": "7.17.10",
+    "@react-theming/storybook-addon": "1.1.5",
     "@rollup/plugin-commonjs": "22.0.0",
     "@rollup/plugin-node-resolve": "13.3.0",
     "@rollup/plugin-typescript": "8.3.2",
+    "@storybook/addon-actions": "6.5.0-beta.8",
```

Comment:
storybook `6.4` doesn't support react 18. `6.5` is in a very good state and will be released soon.